### PR TITLE
Profile avatar

### PR DESCRIPTION
**Pull request by chenyou-H**
go to localhost/3000/componentRender page to see the visual Profile avatar component

### CODE REVIEW

File: components/ProfileAvatar/ContributionAndFollow.tsx
Comment (VijayIyer) [inline components/ProfileAvatar/ContributionAndFollow.tsx:1]: @chenyou-H, in the application, the 3 buttons here are clickable (pointer on hover) and on click, open up different tabs. In your current implementation, i think this is not the case. 
Is that functionality part of this ticket?
If not, I think we should add a ticket in backlog to be picked up later.

**Review by VijayIyer — COMMENTED** [review]
I have started reviewing this pull request, and seems extremely well done.

**Review by VijayIyer — COMMENTED** [review]
@chenyou-H , one minor thing. 
There was a requirement in acceptance criteria to have size of this component based on props, as this is intended to be re-usable component. 
Is that implemented as part of this pull request?

File: components/ProfileAvatar/ContributionAndFollow.tsx
Comment (chenyou-H) [reply components/ProfileAvatar/ContributionAndFollow.tsx:1]: You are right, I miss this functionality. There is no need to add a ticket. I can implement this quick

**Comment by chenyou-H** [comment]
> @chenyou-H , one minor thing. There was a requirement in acceptance criteria to have size of this component based on props, as this is intended to be re-usable component. Is that implemented as part of this pull request?

I also miss that part. Now I added size props to the profile avatar

**Comment by chenyou-H** [comment]
I just implemented those missing functionality, please take a look when you have time.

**Comment by VijayIyer** [comment]
@chenyou-H there is a couple of issues now, as seen in below screenshots. 
- The border/margin doesn't seem to cover the buttons below. image
- When you have 2 of these component instances together vertically, they are overlapping.  Is that per design?
image

**Comment by chenyou-H** [comment]
> @chenyou-H there is a couple of issues now, as seen in below screenshots.
> 
> * The border/margin doesn't seem to cover the buttons below. image
> * When you have 2 of these component instances together vertically, they are overlapping.  Is that per design?
>   image

In order to fix this, I will have to mak

**Pull request closed without merging**